fineract-core/src/main/java/org/apache/fineract/commands/service/SynchronousCommandProcessingService.java

```diff
@@ -272,7 +272,12 @@ private void publishHookEvent(final String entityName, final String actionName,
                 Type type = new TypeToken<Map<String, Object>>() {
 
                 }.getType();
-                Map<String, Object> myMap = gson.fromJson(command.json(), type);
+
```

Comment:
This is not correct... if the provided json is invalid, you should raise a PlatformApiDataValidationException with some message of the nature of the exception, like: "Invalid JSON was provided" or something like this. Feel free to check how the above exception was raised in Fineract and with what parameters.